app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<p><%= "Name: #{@mechanic.name}" %></p>
+
+<p><%= "Years of Experience: #{@mechanic.years_experience}"%>
+
+<section class="assigned-rides">
+  <% @mechanic.rides.each do |ride| %>
+    <p><%= ride.name %></p>
+  <% end %>
+</section>
+
+<p><%= button_to "Add Ride to Mechanic", "/ride_mechanics/new", method: :get, params: {mechanic_id: @mechanic.id} %></p>
```

Comment:
Ahh, ok, I see, you did make a button to get you to the form. That's fine! This button looks good.